web/modules/custom/project_default_content/project_default_content.info.yml

```diff
@@ -4,6 +4,7 @@ package: Custom
 
 type: module
 core: 8.x
+core_version_requirement: ^8||^9
 
 dependencies:
   - default_content:default_content
```

Comment:
As mentioned before, @andypost idea is to get rid of default_content so IMO we should also remove this module until we decide on the new way to import content. We can add it back later if needed.